### PR TITLE
configure meshery ui before building

### PR DESCRIPTION
**Pull request by codeSafari10**
**Notes for Reviewers**

This PR fixes #

add a custom ui config for meshery

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -23,6 +23,16 @@ jobs:
         with:
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+
```
Comment (leecalcote) [inline .github/workflows/build-release.yml:26]: Will this displace any "PLAYGROUND" mode code?

Comment (aabidsofi19) [reply .github/workflows/build-release.yml:26]: @leecalcote  yeah the code in ui that was used to lockdown the navigator and header component is not needed now ( i have removed the code for lockdown of header components )

File: ui/components/User.js [hunk 2]
```diff
@@ -160,6 +161,21 @@ const User = (props) => {
   const { color, iconButtonClassName, avatarClassName, classes } = props;
 
   const open = Boolean(anchorEl);
+
+
+  if ( userData?.status == 'anonymous' ) {
+    const url =
+         `${capabilitiesRegistry?.provider_url}/user/login?anonymousUserID=${userData?.user_id}&redirect=${window.location.pathname}`
```
Comment (leecalcote) [inline ui/components/User.js:168]: @MUzairS15 why is this parameter explicit and not generic like `userID`? This is asymmetric to the extent that there isn't an `ActiveUserID` and a `PendingUserID`, so what justifies breaking the pattern here of now explicitly exposing user account status intermingled with the user ID? If user account status is needed to be known, should it not be exposed as it own `status` parameter given that it _is_ its own property in the schema? Are we being schema-driven in our API endpoint behavior?

**Review by leecalcote — DISMISSED** [review]
Some docs on this enhanced extension point is ideal.

**Comment by leecalcote** [comment]
Once eslint issues are fixed, we can merge. 👍

File: ui/components/User.js [hunk 3]
```diff
@@ -160,6 +161,21 @@ const User = (props) => {
...
+        
+    return (
```
Comment (leecalcote) [inline ui/components/User.js:170]: It'd be great to have a small section in https://docs.meshery.io/extensibility/ui explain the purpose of this extension point.

File: ui/components/User.js [hunk 4]
```diff
@@ -160,6 +161,21 @@ const User = (props) => {
   const { color, iconButtonClassName, avatarClassName, classes } = props;
 
   const open = Boolean(anchorEl);
+
+
+  if ( userData?.status == 'anonymous' ) {
+    const url =
+         `${capabilitiesRegistry?.provider_url}/user/login?anonymousUserID=${userData?.user_id}&redirect=${window.location.pathname}`
+
```
Comment (leecalcote) [inline ui/components/User.js:169]: // EXTENSION POINT: Anonymous Users
// Insert custom logic here to handle Single User mode, Anonymous User mode, Multi User mode behavior.

**Pull request merged**